samples/type_ahead_search_adaptive_cards/csharp/TypeaheadSearch/AppPackage/manifest.json

```diff
@@ -0,0 +1,57 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.typeaheadsearchcard",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Search adaptive card",
+    "full": "Adaptive card with search control"
+  },
+  "description": {
+    "short": "Search control adaptive card",
+    "full": "This sample shows feature type ahead search (static and dynamic) control in adaptive cards."
```

Comment:
This sample shows the feature of type ahead search (static and dynamic) control on Adaptive Cards.